forge/src/main/kotlin/com/github/kittinunf/forge/Forge.kt

```diff
@@ -1,8 +1,6 @@
 package com.github.kittinunf.forge
 
-import com.github.kittinunf.forge.core.Deserializable
-import com.github.kittinunf.forge.core.DeserializedResult
-import com.github.kittinunf.forge.core.JSON
+import com.github.kittinunf.forge.core.*
```

Comment:
🙇 Can you please not do star import?

Reply:
Sure, committed the single name imports instead 👍